packages/@aws-cdk/aws-rds/test/serverless/test.serverless.ts

```diff
@@ -0,0 +1,444 @@
+import { ABSENT, expect, haveResource, haveResourceLike, ResourcePart, SynthUtils } from '@aws-cdk/assert';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import { AuroraPostgresEngineVersion, ServerlessDatabaseCluster, DatabaseClusterEngine, ParameterGroup } from '../../lib';
+
+export = {
+  'can create a Serverless Cluster with Aurora Postgres database engine'(test: Test) {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new ServerlessDatabaseCluster(stack, 'ServerlessDatabase', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      masterUser: {
+        username: 'admin',
+        password: cdk.SecretValue.plainText('tooshort'),
+      },
+      vpc,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).to(haveResource('AWS::RDS::DBCluster', {
```

Comment:
I would implore using `haveResourceLike` more - I don't think we're interested in much beyond `Engine` and `EngineMode` in this test, for example.